### DISCUSSION TITLE
Is it possible to use this control API on the original linux build while the original control app is running?

### DISCUSSION THREAD

**Discussion by churli** [AetherRadio/X6100Control#7] [Q&A]
I would like to play around with writing a couple of command-line tools that detect some conditions of the TRX (band, mode, ptt triggered, etc...) and call some script.
Can this library be used in parallel with the stock control application?

Thanks! :)

**Comment by ruilvo** [comment]
@strijar what do you think? I think probably the stock app and this library will "race" for the serial stream from the BB chip, and it won't work...

**Reply by strijar** [reply]
Yes that's right. It will not work together because I2C and the serial port are already busy.
